shakenfist/virt.py

```diff
@@ -290,9 +291,11 @@ def block_devices(self):
         return self._db_get_attribute('block_devices')
 
     # Implementation
+    def get_lock(self, name, op):
```

Comment:
Isn't this confusing? Its locking an attribute not the entire object?

Reply:
True. Let me modify that.